modules/github-oidc-role/policy_gitops.tf

```diff
@@ -0,0 +1,107 @@
+variable "gitops_policy_configuration" {
+  type        = map(string)
+  default     = {}
+  description = <<-EOT
+    Configuration for the GitOps IAM Policy, valid keys are
+     - `s3_bucket_component_name` - Component Name of where to store the TF Plans in S3, defaults to `gitops/s3-bucket`
+     - `dynamodb_component_name` - Component Name of where to store the TF Plans in Dynamodb, defaults to `gitops/dynamodb`
+     - `s3_bucket_environment_name` - Environment name for the S3 Bucket, defaults to current environment
+     - `dynamodb_environment_name` - Environment name for the Dynamodb Table, defaults to current environment
```

Comment:
Make this an object with explicit defaults. Default to `null` for current environment.